src/utils/flags.ts

```diff
@@ -316,6 +324,11 @@ export const outputFilePathFlag = flags.string({
   required: true,
 })
 
+export const saveFilePathFlag = flags.string({
```

Comment:
Instead of introducing a new flag, let's reuse `outputFilePathFlag`, which is already used by the other export commands.